src/utilities/theme/role-variants.ts

```diff
@@ -599,8 +596,8 @@ export const roleVariants: RoleVariants = {
       name: 'decorativeThreeSurface',
       description: 'For use as a decorative surface color.',
       light: {
-        hue: 144,
-        saturation: 54,
+        hue: (hue) => (((hue + 88) % 360) + 360) % 360,
+        saturation: (sat) => Math.max(sat - 46, 0),
```

Comment:
Since we are only doing subtraction, this prevents that value from being less than 0.